src/detection_linux.cpp

```diff
@@ -89,16 +89,13 @@ void Stop() {
 		return;
```

Comment:
If someone wants to submit a new PR with the review addressed and tested, we can move this forward more. Credit still goes to all involved (@antelle 🙇).

Reply:
Just adding a reference here, too. I think I've made the requested changes: #162